src/pages/docs_sdk_purchase-subscription.html

```diff
@@ -67,6 +67,24 @@ <h3 id="notes">Notes</h3>
     </li>
   </ul>
 
+  <h3 id="advanced">Advanced</h3>
+
+  <ul>
+    <li>You can support calculating tax at checkout. To enable automatic tax:
+      <ul>
+        <li>Add a coupon to your Stripe account under <a href="https://dashboard.stripe.com/coupons" target="_blank">Products > Coupons</a></li>
+        <li>Then add a promotion code for that coupon</li>
+      </ul>
+    </li>
+    <li>You can allow promo codes at checkout. To enable promo codes:
+      <ul>
+        <li>You need to activate tax collection</li>
```

Comment:
Small nit: this is done via the same link as in the next bullet, think it makes sense to include the link in this first bullet instead